### PR TITLE
Added tab completion support for plugin's command arguments

### PR DESCRIPTION
**Pull request by edward720**

### CODE REVIEW

File: src/main/scala/com/tapad/docker/DockerComposePlugin.scala [hunk 1]
```diff
@@ -362,7 +386,7 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
 
       print(s"Inspecting container $containerId to get the port mappings")
       val containerInspectInfo = getDockerContainerInfo(containerId)
-      val jsonInspect = parse(containerInspectInfo)
+      val jsonInspect = net.liftweb.json.parse(containerInspectInfo)
```
Comment (yiming-li-tapad) [inline src/main/scala/com/tapad/docker/DockerComposePlugin.scala]: Hmm.  Why is 'net.liftweb.json' added?  Can we import it instead?

**Comment by kurtkopchik** [comment]
You should add a note to the README that says that tab completion can be used to list out all of the arguments for each command.

File: src/main/scala/com/tapad/docker/DockerComposePlugin.scala [hunk 2]
```diff
@@ -86,12 +87,16 @@ class DockerComposePluginLocal extends AutoPlugin with ComposeFile with DockerCo
   val skipBuildArg = "skipBuild"
   lazy val dockerComposeVersion = getDockerComposeVersion
 
-  lazy val dockerComposeUpCommand = Command.args("dockerComposeUp", ("dockerComposeUp", "Starts a local Docker Compose instance."),
+  val dockerComposeUpRestartArgs = Seq(skipPullArg, skipBuildArg, useStaticPortsArg)
```
Comment (kurtkopchik) [inline src/main/scala/com/tapad/docker/DockerComposePlugin.scala]: I'd prefer this to be named `dockerComposeUpArgs` and used in the restart command vs the opposite which is what you have now.

**Comment by kurtkopchik** [comment]
Thanks @edward720

**Pull request merged**